torchao/quantization/pt2e/observer.py

```diff
@@ -1885,7 +1885,9 @@ def convert(self, model: torch.fx.GraphModule, observer_node: Node):
             )
 
         from torchao.quantization.pt2e.utils import create_getattr_from_value
+        from torchao.utils import _assert_and_get_unique_device
```

Comment:
btw, this is still going to be per node, but here it's probably free to get the device from scale tensor? can we do that

Reply:
oh you're right, let me try